src/network/network_content_gui.cpp

```diff
@@ -539,7 +539,7 @@ class NetworkContentListWindow : public Window, ContentCallback {
 	 *   other types are only shown when content that depend on them are
 	 *   selected.
 	 */
-	NetworkContentListWindow(WindowDesc &desc, bool select_all, const std::bitset<CONTENT_TYPE_END> &types) :
+	NetworkContentListWindow(WindowDesc &desc, bool select_all, ContentTypes types) :
```

Comment:
can't remember what the underlying storage is, maybe pass by ref?

Reply:
`using ContentTypes = EnumBitSet<ContentType, uint16_t, CONTENT_TYPE_END>;`

It's a `uint16_t`.